src/commands/init.ts

```diff
@@ -0,0 +1,33 @@
+import * as fs from 'fs-extra'
+import { join } from 'upath'
+
+import File from '../core/File'
+
+const cliConf: string =`# Kong Admin URL
+# @required
+kong_admin_url: http://localhost:8001
+
+# Kong Admin Token
+# @optional
+kong_admin_token: ''`
+
+export default async (args): Promise<void> => {
+
+  let path = join("workspaces", args.workspace)
+
+  await fs.mkdir(path, { recursive: true }, (err) => {
```

Comment:
Looking at the [`fs-extra` docs](https://github.com/jprichardson/node-fs-extra#sync-vs-async-vs-asyncawait), `fs.mkdir` only returns a promise if a callback is not passed. So I think this should just be `await fs.mkdir(path, { recursive: true })`, or else there will be no result to `await`. And then any error will be thrown automatically.